### ISSUE TITLE
Boards availability. 

### ISSUE THREAD

**Issue by dwhacks** [snietfeld/OpenChronograph#1]
Are you planning on making boards available for purchase? I’d love to tackle this project but don’t really want to order 5x each board and parts to build only one watch. 

Any plans for even a kit or assembled boards or am I on my own. 

Also, the disassembly video, what watch is that? I couldn’t figure it out from the list of “drop in watches”.

**Comment by snietfeld** [comment]
The original watch in the video is a Goodwin hybrid model [FTW1171](https://www.fossil.com/en-us/products/hybrid-smartwatch-goodwin-black-leather/FTW1171.html). That specific model isn't being sold anymore as far as I can tell, but you can still find them on ebay occasionally. If you can't find that, then it looks like the Fossil Neutra line uses the same case design, with the [Neutra Amber](https://www.fossil.com/en-us/products/hybrid-smartwatch-neutra-amber-leather/FTW1194.html) being the closest fit.

I have a small number of kits built up, including mainboard and programmer (but not the watch) available here: https://www.tindie.com/products/oclabs/openchronograph-oc-1-devkit/

**Comment by dwhacks** [comment]
Perfect! I am in Canada so tindie wouldn’t let me purchase and said they have sent you a message about shipping to Canada. Please let me know if that’s possible. Really like this project.

**Comment by snietfeld** [comment]
Ah okay, I just added a flat rate shipping option for Canada--let me know if you have any issues.

**Comment by dwhacks** [comment]
Thanks. Shipping to Canada is more then I would hope so I will have to find another way. I lve close to the US border so I may be able to ship it there and drive across to pick it up. Thanks for the great project.

**Comment by snietfeld** [comment]
Yeah it seemed high to me too, but I'm not super familiar with international shipping. Is there a cheaper carrier or shipping method you'd recommend?

**Comment by dwhacks** [comment]
USPS is definitely the cheapest, and most reliable. I think the problem is the "flat rate international priority". Being Canadian I don't really know what the options are, but for example, I have a radio controlled airplane kit box in my hand that went from Florida to Montreal for $4.90 (printed stamp says price). The box is 3"x6"x30". Another one, $5.75 and says "USPS AIRMAIL".

MadBean pedals will send "flats" for $5 and these are usually a small bubble envelope with a piece of cardboard and the circuit board tapped to it. I have got circuit boards with DIP chips shipped that way.

According to the USPS site, "First-Class Mail International, large envelopes (flats)" is the [cheapest](https://www.usps.com/business/prices.htm), Next up is "First-class package international" starting at $12+

If you're willing to go to a post office with it packaged in a bubble envelope and get a quote I would gladly pay the shipping + packaging costs and could send you paypal directly. Maybe it's tindie that doesn't have a shipping good option?

I'm not trying to be a pain here, or undervalue your time.

e-mail me @ forsakenrider at hotmail dot com

**Comment by snietfeld** [comment]
Thanks for providing the other postage prices for comparison--glad to see there are cheaper methods out there. I'm out of town this weekend, but I'll make a run to the post office when I get back to find out what the options are.

**Comment by dwhacks** [comment]
excellent, thank you!
